change log entry for CIE-5609 (#4717)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2025.0.15-logging-operator-pod-crash-loop.md

---
date: 2024-01-16
title: Logging operator remains stable after Edge upgrade
change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5609
version: 2025.0.15
---

> Fixed an issue where the logging operator pod could crash after upgrading Edge to `2025.0.12` or later. The logging operator now remains stable after the upgrade.

Reviewer: ```suggestion
Previously, the logging operator pod could crash after upgrading Edge to version `2025.0.12` or later. This issue has been fixed. The logging operator now remains stable after the upgrade.
```
